### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Gruber Adam, Pittet Axel

### DISCUSSION THREAD

**Discussion by Axwells** [heig-vd-dai-course/heig-vd-dai-course#495] [Show and tell]
Our idea is to create a chat that contains different group chats (from 1 to 5 people). We know that a lot of groups will do something similar to our project and this is why we want to extend it by adding a chat history.

The history will be stored in a text file that could be loaded if the user asks for it.

The functionnalities are the following :
- User can choose a group chat among which he has access to
- User can write messages in these group chats
- User can receive messages from other users in these group chats
- User can load the history of the group chat he's in

**Comment by ludelafo** [comment]
Looks good to us!

**Comment by AdamoElProfesor** [comment]
The project is done.
- **Repository Link** : https://github.com/AdamoElProfesor/dai_practical_work_2
- **Commit Hash** : f7626468aef1b36179d849fd3c8b9e85d45a35d9
